ISISComputingGroup/EPICS-DeviceEmulator
# Ticket 5266: Reconnect daqmx on error

See https://github.com/ISISComputingGroup/IBEX/issues/5266. Added some VIs for starting and stopping a simulated NI MAX

To test:
* Open NI MAX
* Run `start_sim.bat`
* Confirm a new device and associated task appear as per http://www.ni.com/tutorial/3698/en/
* Run `stop_sim.bat` and confirm it disappears
* Open `DAQmx Simulator.lvproj` and confirm you can rebuild the exe
